openmoc/map_to_dict.i

```diff
@@ -107,15 +107,15 @@
     return NULL;
   }
 
-  $1 = PySequence_Length($input);  // num_x
-  $2 = PySequence_Length(PyList_GetItem($input,0)); // num_y
+  $1 = PySequence_Length($input);  // num_y
```

Comment:
There was just an inconsistency with the template for the `setUniverses` method. The template for the method exists in three places: `map_to_dict.i`, 'Universe.h`, and `Universe.cpp`. In `map_to_dict.i` and `Universe.h` the template was:

`(int num_x, int num_y, Universe** universes)`

In `Universe.cpp` the template was:

`(int num_y, int num_x, Universe** universes)`

Note the switch of `num_x` and `num_y` in the templates. These modifications make all the templates consistent. The previous templates produced the desired result; their was just an inconsistency that could cause confusion.